ADA/ada.qmd

```diff
@@ -19,57 +17,234 @@ It brings together:
+| Benefits out-of-the-box (minimal configuration) | Yes | No |
+| Deterministic outputs (i.e. same result every run) | No | Yes |
+| Supports modular code structure & packages | No | Yes |
+| Supports unit testing frameworks | No | Yes |
+| Nice version control (e.g. readable git diffs) | No | Yes |
+| Autocomplete / auto-formatting & syntax highlighting | Yes* | Yes |
+| Compatible with sharing code externally | Yes** | Yes |
+  
+*= auto-formatting is available for SQL and Python in Databricks, but not R  
+**= external users must have software that would allow them to open notebooks
+
+Notebooks can be helpful for ad hoc, exploratory, or one off pieces of analysis to add additional documentation as you go, but please consider the skill sets and knowledge levels in your team when doing so. You can also use notebooks to present findings to others by hiding code cells and only showing Markdown cells containing charts, tables, images or text. They can be exported as HTML files to enable easier sharing outside of the Databricks interface. 
+
+For regular analysis pipelines adopting RAP principles we'd usually advise against using notebooks and instead recommend a Git controlled repository of code scripts using the [standard repository template available in dfeR](https://dfe-analytical-services.github.io/dfeR/reference/create_project.html). This is because version control can be more difficult in Databricks notebooks, as there is the potential for multiple people to make edits to the same notebook at once. Additionally, with a full script in an IDE, when the code is run without modifications it will provide the same output consistently. However, with notebooks you can run code cells independently of one another. If you don't run all code cells in order then your output could be inconsistent and result in errors, potentially making QA more difficult. 
+
+
+:::{.callout-warning}
+When using Databricks notebooks with git, cell outputs are automatically cleared by default when your code is pushed up to a repository. However, if you share a notebook using the Databricks interface, cell outputs remain intact from the last time you ran the code. This risks sharing data that should not be shared with others. Before sharing code with someone outside your team, you must use the `Clear all cell outputs` option in the Databricks "Run" menu to avoid inadvertently sharing data that only you and your team should have access to. 
+:::
+
+If the rest of your team are unfamiliar with notebooks and will need to pick up your work at a later date, this could cause issues. The [Statistics Development Team](mailto:statistics.development@education.gov.uk) are happy to advise on this and provide guidance if you have any concerns. 
+
+
+You can read more about the differences between IDEs and notebooks and where each might be useful [on the NHS-R website](/ADA/ada.html#differences-between-ides-and-notebooks).
+
+---
+
+#### Writing and running code: scripts and IDEs
```

Comment:
Can we move this scripts / IDEs section above the notebooks section? It's the default way we'd expect and is what most teams are already more familiar with